packages/cli/src/helpers.mts

```diff
@@ -10,6 +10,14 @@ import { has } from 'lodash-es';
 
 const DIR_NAME = getDirNameForCurrentFile(import.meta);
 
+// helpers when when working with url to generate service
+const DEFAULT_BRANCH = 'develop';
+const URL_PREFIX = 'https://api.github.com/repos/harness/harness-core/contents/';
+
+export const GITHUB_PAT = process.env.GITHUB_PAT;
+export const GITHUB_API_ENDPOINT_URL = (yamlPath: string) =>
+	`${URL_PREFIX}${yamlPath}?ref=${DEFAULT_BRANCH}`;
```

Comment:
Any other way to generate the path—where yaml path itself has context of branch, instead of passing it as a query param?